lib/components/molecules/StickyBar/StickyBar.story.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+import defaultConfig from './StickyBar.mock';
+
+// Import Styled Component to showcase variations
+import StickyBar, { StickyBarVanilla } from '.';
+
+// TODO: indentation is not working fine in editor. Need to check for .hbs files
+storiesOf('Molecules/StickyBar', module)
+  .addParameters({ jest: ['StickyBar', 'StickyBarVanilla'] })
+  .add('Knobs', () => (
+    <StickyBarVanilla
+      {...defaultConfig}
+      className={`hide-default-sample ${defaultConfig.className}`}
+    />
+  ))
+  .add('StickyBar', () => (
+    <StickyBar {...defaultConfig}>
+      <StickyBar.Item>List Element 1</StickyBar.Item>
```

Comment:
Instead of List Element 1 and dummy data make an example of social bar by using Button and Icons